src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/model/Envelope.java

```diff
@@ -24,6 +25,7 @@
     public Envelope(
         @JsonProperty(value = "id", required = true) String id,
         @JsonProperty(value = "case_ref", required = true) String caseRef,
+        @JsonProperty("previous_service_case_ref") String previousServiceCaseRef,
```

Comment:
`required=false` for clarity?

Reply:
Idea complains about redundant parameter if I do

Reply:
people > intellij ;)